src/assets/default.scss

```diff
@@ -1,4 +1,4 @@
-@import url("https://fonts.googleapis.com/css?family=Lato|Merriweather"); // sass-lint:disable-line no-url-domains, no-url-protocols
+@import url('https://fonts.googleapis.com/css?family=Lato|Merriweather'); // sass-lint:disable-line no-url-domains, no-url-protocols
```

Comment:
@chingsley why was this rule changed?

Reply:
The Eslint and hound rules were contradictory. Eslint would flag double quotes as errors locally, but when we push our code, Hound would flag single quotes as errors. As a result, we made the changes  so that the two rules (Esling and hound) are aligned